spec/models/game_spec.rb

```diff
@@ -36,6 +36,19 @@
     end
     
   end
-  
-  
+
+  describe "player's turn" do 
+    let(:white_player) { FactoryGirl.create(:player, email: 'blah@blah.com', password: 'SPACECAT') }
+    let(:black_player) { FactoryGirl.create(:player, email: 'meow@meow.com', password: 'MONORAILCAT') }
+    let(:game) { FactoryGirl.create(:game, white_player_id: white_player.id, black_player_id: black_player.id) }
+    let(:piece) { FactoryGirl.create(:piece, game_id: game.id, player_id: white_player.id) }
+
+    it "should change turns after player moves" do
+    expect(game.set_default_turn).to eq(true)
```

Comment:
Yes, @kellylougheed - you're absolutely right - I tested it with byebug and it should equal white player id. I'll change it and hope it does equal the white player instead of just being true.